client/js/isaac/interface/visualisation.js

```diff
@@ -1115,4 +1117,37 @@ function updateWireframe() {
 		255 - parseInt(background_color.jscolor.rgb[2]));
 	let enabled = document.getElementById("wireframe_checkbox").checked;
 	renderer.setWireframe(enabled, color);
+}
+
+/**
+ * 
+ * @param {IsaacResponse} response 
+ */
+function aoSetValues(response) {
+	
+	if (response.hasOwnProperty("ao isEnabled")) {
```

Comment:
just wondering: are spaces in that key a good idea?

Reply:
Typically you would use ao as extra class and isEnabled and weight as it's attributes 
`{
    "ao": {
             "isEnabled": true, 
             ...
             }
}`
but i was unable create extra classes in janson (they were never attached) so i kept this style and would change this if the json overhaul with a more c++ like lib begins (there are some other attributes that have white spaces and are awful to deal with in js like response["test 123"] instead of response.test_123)

So basically yes its a bad style and should not be in the codebase.

Reply:
Yes I also vote for using a modern json API.